collections/triple.go

```diff
@@ -273,6 +273,15 @@ func (t tripleKeyCodec[K1, K2, K3]) SizeNonTerminal(key Triple[K1, K2, K3]) int
 	return size
 }
 
+// PrefixUntilTripleRange defines a collection query which ranges until the provided Pair prefix.
```

Comment:
You may want to split this in a separate PR, so that you can use a pseudo version from main.

Reply:
Yeah